src/net/socket.rs

```diff
@@ -38,6 +38,14 @@ impl Socket {
         Socket::bind_with_config(addresses, Config::default())
     }
 
+    /// Bind to any local port on the system, if available
+    pub fn bind_any() -> Result<(Self, Sender<Packet>, Receiver<SocketEvent>)> {
```

Comment:
Probably want a version of this which takes a Config

Reply:
Done